src/wasm-features.h

```diff
@@ -145,6 +154,11 @@ struct FeatureSet {
     return (features & SharedEverything) != 0;
   }
   bool hasFP16() const { return (features & FP16) != 0; }
+  bool hasBulkMemoryOpt() const {
+    bool has_bmo = (features & (BulkMemoryOpt)) != 0;
+    assert(has_bmo || !hasBulkMemory());
+    return has_bmo;
```

Comment:
```suggestion
    bool has = (features & (BulkMemoryOpt)) != 0;
    assert(has || !hasBulkMemory());
    return has;
```
Seems like it is clear what we have here. Or, we could do `has_bmo => hasBMO` to follow the conventions elsewhere.